Apps/Patient/src/Properties/launchSettings.json

```diff
@@ -4,26 +4,18 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:3002"
+      "applicationUrl": "https://localhost:3002"
     }
   },
   "profiles": {
-    "IIS Express": {
-      "commandName": "IISExpress",
-      "launchBrowser": true,
-      "launchUrl": "swagger",
-      "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
-      }
-    },
-    "template": {
+    "HealthGateway.Patient": {
       "commandName": "Project",
       "launchBrowser": true,
-      "launchUrl": "api/values",
+      "launchUrl": "swagger",
```

Comment:
not 100% sure we want this. Most of the time we are not using swagger when lunching for dev

Reply:
This will at least launch some UI when debugging the sevices instead of a blank page